app/src/main/res/layout/activity_main.xml

```diff
@@ -2,33 +2,51 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data class="com.mycompany.myapp.ui.main.MainActivityBinding">
+    <data>
         <variable
             name="vm"
             type="com.mycompany.myapp.ui.main.MainViewModel"/>
     </data>
 
-    <androidx.coordinatorlayout.widget.CoordinatorLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I think we will still want to leave a coordinatorlayout at the top for things like a nav drawer? Especially if the toolbar still lives in the activity? (Thinking out loud...not sure the correct answer)

Reply:
So tried moving the toolbar into the MainFragment instead of the activity. So we still have coordinator layout in the activity? Will be in next push for us to look at.